src/components/DisplayArticles.jsx

```diff
@@ -27,4 +27,5 @@ function DisplayArticles() {
     )
 }

```

Comment:
Line 11 take out console logs before any PR or merging!

Reply:
Lines 8 - 14

You can make the anonymous function on line 7 async so you don't need to wrap what's happening in the useeffect in another function 
``` js
useEffect(async () =>
{
    // my implementation
    await MyMethodAsync(...);
});
```

I would take line 8 and line 14 out of the effect

I'd also consider putting a try catch block in to keep the error handling in here (the end of the promise chain) rather than doing catching in the api.js file

Reply:
Line 17 - please add a Loading pattern to show a user when data is being loaded and the site isn't broken.